### DISCUSSION TITLE
VS Code on Atlas OS

### DISCUSSION THREAD

**Discussion by DesejadoMoamed** [Atlas-OS/Atlas#1196] [Help & Support] [answered]
Can anyone help with this problem, I can't open settings VScode on Atlas Os.

I always get this error when I open settings,
![Screenshot 2024-08-27 180603](https://github.com/user-attachments/assets/559a0f52-a8c1-4417-b6bb-9972f46a8e0e)

And when I try installing new extension it gives me this error, 
![Screenshot 2024-08-27 180743](https://github.com/user-attachments/assets/13b9726d-e574-4849-bbd4-7c649640f550)
Downloading it mannually doesn't change anything.

I tried looking everywhere, but can't find the solution.

**Comment by he3als** [comment, marked as answer]
Your problem is unrelated to Atlas. Your VSCode configuration is a renamed PNG file instead of a JSON; try resetting or deleting your VSCode data.

**Reply by DesejadoMoamed** [reply]
Thanks for clarifying that.
